test/types/PoolKey.t.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {PoolId, PoolIdLibrary, PoolKey} from "../../src/types/PoolId.sol";
+
+contract TestPoolKey is Test {
+    using PoolIdLibrary for PoolKey;
+    
+    function testPoolIdCreation(PoolKey memory poolKey) public {
```

Comment:
I think this file isnt actually needed - it does the same as `PoolId.t.sol`

Reply:
Then I can merge into our temp branch :)

Reply:
done in [47cf6e3](https://github.com/Uniswap/v4-core/pull/550/commits/47cf6e3353184eea305132e8ebd13c52e47e0237)